pkgs/development/libraries/cpp-utilities/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "cpp-utilities-${version}";
+  version = "4.17.1";
+
+  src = fetchFromGitHub {
+    owner = "Martchus";
+    repo = "cpp-utilities";
+    rev = "v${version}";
+    sha256 = "12088cwg3jbqipmbn4843w1cgxi1q6vwx47gy042rkfvbk6azhxl";
+  };
+
+  outputs = [ "out" "dev" ];
+  doCheck = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/Martchus/cpp-utilities;
+    description = "Common C++ classes and routines used by my applications such as argument parser, IO and conversion utilities";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ doronbehar ];
+  };
```

Comment:
`platforms` is missed.

Reply:
I noticed `platforms` is not always given to the `meta` attribute, is it needed here because it's a library?

Reply:
Good question! I am not sure either.
Hello, @marsam Could you help us? Could you please give us some suggestions?
Thank you very much!!

Reply:
I think `platforms` should be present to allow [hydra have prebuilts](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes)
>Hydra builds packages according to the platform specified. If no platform is specified, the package does not have prebuilt binaries.

Reply:
Good to know, Thanks!